src/backend/opengl/TextureGL.cpp

```diff
@@ -38,9 +38,11 @@ namespace backend { namespace opengl {
                     return {GL_RGBA8, GL_RGBA, GL_UNSIGNED_BYTE};
                 case nxt::TextureFormat::R8G8B8A8Uint:
                     return {GL_RGBA8UI, GL_RGBA, GL_UNSIGNED_INT};
+                case nxt::TextureFormat::R8Unorm:
+                    return {GL_R8, GL_RED, GL_UNSIGNED_BYTE};
                 case nxt::TextureFormat::B8G8R8A8Unorm:
-                    // This doesn't have an enum for the internal format in OpenGL.
-                    return {GL_NONE, GL_BGRA, GL_UNSIGNED_BYTE};
+                    // This doesn't have an enum for the internal format in OpenGL, so use RGBA8.
```

Comment:
Can we have this fix in a separate commit?

Reply:
Done.